docs/DevGuide/WritingTests.md

```diff
@@ -194,3 +194,20 @@ the `ifdef SPECTRE_DEBUG` block, they can just call have the code that triggers
 an `ERROR`. For example,
 
 \snippet Test_AbortWithErrorMessage.cpp error_test_example
+
+### Building and Running A Single Test File
+
+In cases where low-level header files are frequently being altered and the
+changes need to be tested, building `RunTests` becomes extremely time
+consuming. The `RunSingleTest` executable in `tests/Unit/RunSingleTest` allows
+one to compile only a select few of the test source files and only link in the
+necessary libraries. To set which test file and libraries are linked into
```

Comment:
If it's a select few, and not just one, then shouldn't this be "RunOnlyTheseTests" or something, instead of "RunSingleTest"?

Reply:
The point of the executable is to be able to run a single test without recompiling the entire tree. I'm leaving the name as is.